cloudbuild.yaml

```diff
@@ -73,11 +73,13 @@ steps:
     entrypoint: bash
     args:
       - "-e"
-      - "./scripts/publish-functions.sh" 
-      - "$PROJECT_ID" 
-      - "$_BUCKET" 
-      - "$_KEY_LOCATION" 
-      - "$_KEY_RING" 
+      - "./scripts/publish-functions.sh"
+      - "$PROJECT_ID"
+      - "$_BUCKET"
+      - "$_KEY_LOCATION"
+      - "$_KEY_RING"
       - "$_FUNCTION_REGION"
 # Each function can take ~2 minutes to deploy.
-timeout: 2400s
+# This means at 5 functions, we will run out
+# at the 10 minute mark. Increase to 20.
+timeout: 1200s
```

Comment:
just a bad merge?

Reply:
Crud, yes.  Apologies!  No idea why this even changed.